control/autoware_smart_mpc_trajectory_follower/autoware_smart_mpc_trajectory_follower/scripts/proxima_calc.cpp

```diff
@@ -215,45 +253,58 @@ class transform_model_to_eigen
     acc_ctrl_queue_size = acc_ctrl_queue_size_;
     steer_ctrl_queue_size = steer_ctrl_queue_size_;
     steer_ctrl_queue_size_core = steer_ctrl_queue_size_core_;
+    vel_normalize = vel_normalize_;
+    acc_normalize = acc_normalize_;
+    steer_normalize = steer_normalize_;
   }
   Eigen::VectorXd error_prediction(Eigen::VectorXd x)
   {
     Eigen::VectorXd acc_sub(acc_ctrl_queue_size + 1);
-    acc_sub[0] = x[1];
-    acc_sub.tail(acc_ctrl_queue_size) = x.segment(3, acc_ctrl_queue_size);
+    acc_sub[0] = acc_normalize * x[1];
+    acc_sub.tail(acc_ctrl_queue_size) = acc_normalize * x.segment(3, acc_ctrl_queue_size);
 
     Eigen::VectorXd steer_sub(steer_ctrl_queue_size_core + 1);
-    steer_sub[0] = x[2];
+    steer_sub[0] = steer_normalize * x[2];
     steer_sub.tail(steer_ctrl_queue_size_core) =
-      x.segment(3 + acc_ctrl_queue_size, steer_ctrl_queue_size_core);
-    Eigen::VectorXd acc_layer_1 = relu(weight_0 * acc_sub + bias_0);
+      steer_normalize * x.segment(3 + acc_ctrl_queue_size, steer_ctrl_queue_size_core);
+    Eigen::VectorXd acc_layer_1 = relu(weight_acc_layer_1 * acc_sub + bias_acc_layer_1);
```

Comment:
```suggestion
    const Eigen::VectorXd acc_layer_1 = relu(weight_acc_layer_1 * acc_sub + bias_acc_layer_1);
```
can we add const? plase add const as much as possible for .cpp